packages/kit/src/runtime/app/forms.js

```diff
@@ -117,6 +117,11 @@ export function enhance(form_element, submit = () => {}) {
 			: clone(form_element).method;
 		if (method !== 'post') return;
 
+		const skip =
```

Comment:
I wonder if we can reuse the `get_router_options` function to get the reload option state
https://github.com/sveltejs/kit/blob/9a4a41170ab3e17f947eb4a67944958b22da4236/packages/kit/src/runtime/client/utils.js#L151